packages/account-abstraction/src/HttpRpcClient.ts

```diff
@@ -87,7 +87,7 @@ export class HttpRpcClient {
   async estimateUserOpGas(userOp1: Partial<UserOperation>): Promise<string> {
     // await this.initializing
     const hexifiedUserOp = deepHexlify(await resolveProperties(userOp1))
-    const jsonRequestData: [UserOperation, string] = [hexifiedUserOp, this.entryPointAddress]
```

Comment:
t's not used anywhere? also in the reference aa sdk?

Reply:
same as above